### ISSUE TITLE
[PHP 5.x / ZTS] failed build

### ISSUE THREAD

**Issue by remicollet** [grpc/grpc#12114]
```
/usr/include/php-zts/php/Zend/../TSRM/TSRM.h:164:57: error: 'tsrm_ls' undeclared (first use in this function)
 #define TSRMG(id, type, element) (((type) (*((void ***) tsrm_ls))[TSRM_UNSHUFFLE_RSRC_ID(id)])->element)
                                                         ^
/usr/include/php-zts/php/Zend/zend_hash.h:110:28: note: in definition of macro 'zend_hash_update'
   _zend_hash_add_or_update(ht, arKey, nKeyLength, pData, nDataSize, pDest, HASH_UPDATE ZEND_FILE_LINE_CC)
                            ^~
/builddir/build/BUILD/php-pecl-grpc-1.4.3/ZTS/src/php/ext/grpc/channel.c:188:3: note: in expansion of macro 'PHP_GRPC_PERSISTENT_LIST_UPDATE'
   PHP_GRPC_PERSISTENT_LIST_UPDATE(&EG(persistent_list), key, key_len,
   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/include/php-zts/php/Zend/zend_globals_macros.h:45:16: note: in expansion of macro 'TSRMG'
 # define EG(v) TSRMG(executor_globals_id, zend_executor_globals *, v)
                ^~~~~
/builddir/build/BUILD/php-pecl-grpc-1.4.3/ZTS/src/php/ext/grpc/channel.c:188:36: note: in expansion of macro 'EG'
   PHP_GRPC_PERSISTENT_LIST_UPDATE(&EG(persistent_list), key, key_len,
                                    ^~
/usr/include/php-zts/php/Zend/../TSRM/TSRM.h:164:57: note: each undeclared identifier is reported only once for each function it appears in
 #define TSRMG(id, type, element) (((type) (*((void ***) tsrm_ls))[TSRM_UNSHUFFLE_RSRC_ID(id)])->element)
                                                         ^
/usr/include/php-zts/php/Zend/zend_hash.h:110:28: note: in definition of macro 'zend_hash_update'
   _zend_hash_add_or_update(ht, arKey, nKeyLength, pData, nDataSize, pDest, HASH_UPDATE ZEND_FILE_LINE_CC)
                            ^~
/builddir/build/BUILD/php-pecl-grpc-1.4.3/ZTS/src/php/ext/grpc/channel.c:188:3: note: in expansion of macro 'PHP_GRPC_PERSISTENT_LIST_UPDATE'
   PHP_GRPC_PERSISTENT_LIST_UPDATE(&EG(persistent_list), key, key_len,
   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/include/php-zts/php/Zend/zend_globals_macros.h:45:16: note: in expansion of macro 'TSRMG'
 # define EG(v) TSRMG(executor_globals_id, zend_executor_globals *, v)
                ^~~~~
/builddir/build/BUILD/php-pecl-grpc-1.4.3/ZTS/src/php/ext/grpc/channel.c:188:36: note: in expansion of macro 'EG'
   PHP_GRPC_PERSISTENT_LIST_UPDATE(&EG(persistent_list), key, key_len,
                                    ^~
/builddir/build/BUILD/php-pecl-grpc-1.4.3/ZTS/src/php/ext/grpc/channel.c: At top level:
cc1: error: unrecognized command line option '-Wno-parentheses-equality' [-Werror]
cc1: all warnings being treated as errors
```

**Comment by anoopmv** [comment]
HI,

While running grpc hello world program I am facing this issue, please help
me in this:

Traceback (most recent call last):
  File "greeter_client.py", line 48, in <module>
    run()
  File "greeter_client.py", line 43, in run
    response = stub.SayHello(helloworld_pb2.HelloRequest(name='you'))
  File "/usr/local/lib/python2.7/dist-packages/grpc/_channel.py", line 507,
in __call__
    return _end_unary_response_blocking(state, call, False, deadline)
  File "/usr/local/lib/python2.7/dist-packages/grpc/_channel.py", line 455,
in _end_unary_response_blocking
    raise _Rendezvous(state, None, None, deadline)
grpc._channel._Rendezvous: <_Rendezvous of RPC that terminated with
(StatusCode.UNAVAILABLE, Connect Failed)>



Regards,
Anoop

**Comment by remicollet** [comment]
@anoopmv your problem is cleary unrelated to this issue

**Comment by stanley-cheung** [comment]
Thanks for the bug report and the PR. I am fixing a few other issues with 1.4.3 in #12108.

**Comment by stanley-cheung** [comment]
Fixed in 1.4.4
